### ISSUE TITLE
Here's mine redacted, but I couldn't get it to display.

### ISSUE THREAD

**Issue by kateortman** [twgardner2/navy_psr#21]
[PSRreport_Redacted.pdf](https://github.com/twgardner2/navy_psr/files/7921403/PSRreport_Redacted.pdf)

**Comment by twgardner2** [comment]
Kate - thank you! I think the reason it's not rendering is that the redaction "flattened" the file, i.e. turned it into an image. You'll notice that you can't click/drag to select text, like you should be able to on your unredacted PSR. 

If you're comfortable trying it, you could try processing your unredacted PSR on the site (there is certainly no reason to send it to me, I would just be curious to check that it works).

**Comment by twgardner2** [comment]
In case that didn't make sense, this is what I mean:
![flattened_pdf](https://user-images.githubusercontent.com/26434599/150698046-cd03e87e-321f-414a-994a-d4a155b6b82b.gif)

**Comment by kateortman** [comment]
Ah, that makes sense. I'll try again on my end. You don't need more
redacted PSRs for testing?

**Comment by kateortman** [comment]
The unredacted PSR worked! Thanks!

**Comment by twgardner2** [comment]
That's great. I just saw your email about getting more PSRs. I think it's a great idea - I replied to your gmail.

**Comment by ChaseWPDEV** [comment]
@kateortman @twgardner2  Super excited to see you two tinkering in the PSR Parser. A couple of notes/ points of clarification as we "evangelize" the charts (you both probably already know this, stuff). I eventually want to add these as FAQ's

- _How is this secure?_ **No PDF information is transmitted to our server** All PDF parsing occurs in your browser. If you're really curious, go ahead and open the network panel. You'll see nothing goes over the wire.

- _What if your site gets hacked?_ We don't receive or retain any visitor data on the site so there's nothing to steal.

- _What if someone's listening without you knowing?_ Check the SSL lock in your browser. If the lock's green, no external sources are loading and you can be sure it's our site. If it's not green, STOP, do not upload your PDF and contact us.

- _I really don't trust you, but I want a report._  No problem it's just gonna be a little more work, you'll need to use the interactive table below and manually enter the data off your report. If you want to be really cagey, you can even use imaginary names for your Reporting Seniors (I recommend [characters from Under Siege](https://www.imdb.com/title/tt0105690/fullcredits/?ref_=tt_ql_cl),  but it's your choice).

**Comment by twgardner2** [comment]
Great points, Chase. I spend a lot of time this weekend explaining what "client-side" means. It was a popular joke to crack that we are sucking up everyone's PII, although I'm not sure to what end :)

I got some good feedback and was able to see how effectively this visualizes someone's record. Bobby Thompson ran his PSR through it and I could instantly understand that his record is one that belongs to a rockstar. I'm going to submit some issues to record the feedback I got and for us to discuss/prioritize.
